### PR TITLE
Haf plane pop out

### PR DESCRIPTION
**Pull request by lexihaberl**

### CODE REVIEW

**Comment by jibweb** [comment]
Does this need a full on review, or are you confident enough to merge it as is?

**Comment by lexihaberl** [comment]
The changes are not that big, so I will just go through them with David I guess.

**Review by jibweb — APPROVED** [review]

**Comment by jibweb** [comment]
> The changes are not that big, so I will just go through them with David I guess.

I approved the request, so you can merge when you feel confident. I trust you checked it works :-)

File: .gitignore [hunk 1]
```diff
@@ -0,0 +1 @@
+﻿*.pyc
```
Comment (haberger) [inline .gitignore:1]: Sehr gute Arbeit, der code liest sich sehr verständlich. Weiter so <3

File: scripts/statemachine.py [hunk 2]
```diff
@@ -269,12 +275,13 @@ def execute(self, userdata):
             elif char_in == states_keys[States.FIND_GRASP]:
                 rospy.loginfo('Finding a grasp pose')
                 print('Choose a method for grasppoint calculation')
-                print('\t1 - unknown object')
-                print('\t2 - known object')
-                print('\t3 - choose known object')
+                print('\t1 - Detectron unknown object')
```
Comment (haberger) [inline scripts/statemachine.py:278]: Code redundancy could put menu print in separate  function.

**Review by haberger — APPROVED** [review]

**Pull request merged**